### ISSUE TITLE
Apply Best Guess for Deck Names

### ISSUE THREAD

**Issue by coast2coasting** [cderickson/MTGO-DB#1]
I have a question about an error I'm getting, ever since Dec 1, the 'Apply Best Guess for Deck Names' stopped working, it names everything 'Unknown'. I reinstalled and loaded my gamelogs from scratch, ran the guesser, and it worked only for matches pre-Dec 1,
<img width="752" alt="Screenshot 2023-12-17 015905" src="https://github.com/cderickson/MTGO-DB/assets/8650338/c10b5e0b-e81d-4a2e-93d3-005e4b6c9ea0">

**Comment by cderickson** [comment]
It's not working because I haven't scraped decklist data for November yet (Matches played in December use November data to guess deck names).

Normally I do this on the first of every month but I've been busy this month. I'll try to update date it tomorrow (Sunday).

After I update for November, you can go to: Data -> Update Auxiliary Files, then the Best Guess function should work appropriately.

**Comment by coast2coasting** [comment]
I see, could the code be updated to use the latest available months data
until a new month is upload?

**Comment by cderickson** [comment]
Yeah that's something I could think about doing, adding an extra month would make the deck names less accurate but at least it would give you something.

Also I just updated the decklist data to include November, so it should work now after you Update Auxiliary Files.

Let me know if you have any other questions

**Comment by coast2coasting** [comment]
Thank you, works great again. I appreciate the quick response. I also noticed that Modern Yawgmoth decks and Modern Living End decks often have a hard time picking up a deck name.

Your tool is fantastic! Thanks again.
